Debian astroid 2.4.2: ExceptionModelTest.test_oserror

Hello,
while preparing 2.4.1 for inclusion in Debian, the test suite generated this failure:

```
_______________________ ExceptionModelTest.test_oserror ________________________

self = <tests.unittest_object_model.ExceptionModelTest testMethod=test_oserror>

    def test_oserror(self):
        ast_nodes = builder.extract_node(
            """
        try:
            raise OSError("a")
        except OSError as err:
           err.filename #@
           err.filename2 #@
           err.errno #@
        """
        )
        expected_values = ["", "", 0]
        for node, value in zip(ast_nodes, expected_values):
            inferred = next(node.infer())
>           assert isinstance(inferred, astroid.Const)
E           AssertionError: assert False
E            +  where False = isinstance(Uninferable, <class 'astroid.node_classes.Const'>)
E            +    where <class 'astroid.node_classes.Const'> = astroid.Const

tests/unittest_object_model.py:586: AssertionError
=== 1 failed, 880 passed, 35 skipped, 14 xfailed, 1 xpassed in 10.17 seconds ===
```

--- a/tests/unittest_object_model.py
+++ b/tests/unittest_object_model.py
@@ -20,6 +20,13 @@ import astroid
 from astroid import builder, nodes, objects, test_utils, util
 from astroid.exceptions import InferenceError
 
+try:
+    import six  # pylint: disable=unused-import
+
+    HAS_SIX = True
+except ImportError:
+    HAS_SIX = False
+
 
 class InstanceModelTest(unittest.TestCase):
     def test_instance_special_model(self) -> None:
@@ -566,6 +573,7 @@ class ExceptionModelTest(unittest.TestCase):
         inferred = next(ast_node.infer())
         assert isinstance(inferred, astroid.Const)
 
+    @unittest.skipIf(HAS_SIX, "This test fails if the six library is installed")
     def test_oserror(self) -> None:
         ast_nodes = builder.extract_node(
             """

